Separated introduction section of MXD tutorial (#95)
eclipse-tractusx/tutorial-resources: mxd/1_MXD_Introduction.md

# 1 Introduction

> With the Minimum Tractus-X Dataspace, IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.

Reviewer: ```suggestion
With the Minimum Tractus-X Dataspace, IT departments can set up their own data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
```

> With the Minimum Tractus-X Dataspace, IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.

Reviewer: ```suggestion
With the Minimum Tractus-X Dataspace (MXD), IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
```

> For whom is that relevant?

Reviewer: ```suggestion
## 0. Intended audience
```

> This tutorial is designed for companies that want to perform data exchange in a "real" Catena-X data space infrastructure.

Reviewer: That is not quite true.
```suggestion
This tutorial is designed for developers who want to get their hands dirty, for companies that want to perform data exchange in actual Catena-X data space infrastructure and generally for curious minds who want to explore dataspaces.
```

> Who should execute the tutorial?

Reviewer: move to intended audience. And you don't "execute" a tutorial. You "read" it, or fwiw you "do" it.

> IT-Employees with the following skills and previous experience:

Reviewer: ```suggestion
Required knowledge/skills:
```

- Beginner level docker and kubernetes
- Beginner level terraform
- Basic linux system commands

## 1.1 Components & Architecture

By performing this tutorial a data space will be set up including the following components:

- 2 EDC Connectors (Called Bob and Alice)

> - 1 Managed Identity Wallet

Reviewer: explain briefly what these components are. people may not know

- 1 Keycloak instance
- 1 Postgres data base

## 1.2 Prerequisites

> In order to run the Minimum Tractus-X Dataspace "MXD" on your local machine, please make sure the following

Reviewer: ```suggestion
In order to run the Minimum Tractus-X Dataspace on your local machine, please make sure the following
```
